server/mongodb/actions/Analytics.ts

```diff
@@ -123,3 +124,256 @@ export const updateSessionComplete = async (
 
   return data;
 };
+
+// average weekly stats function
+export const averageWeeklyStats = async () => {
```

Comment:
Update the function header's return type to `Promise<null>` to satisfy type checks